layouts/partials/header.html

```diff
@@ -34,6 +34,7 @@
   {{ $styles := resources.Get "css/docs.css" | fingerprint }}
 
   <link rel="stylesheet" href="{{ $styles.Permalink }}" integrity="{{ $styles.Data.Integrity }}">
+  <link rel="canonical" href="{{ .Permalink | replaceRE `(https:\/\/([^\/]*)\/manual\/)([^\/]*\/)(.*)` `$1$4` }}" />
```

Comment:
Regex notes:

* I included `https://` at the beginning because I think we only care that this URL is accurate in production.
* I did not hardcode the production domain (docs.camunda.org) because I thought it would feel better if the staging URLs were canonicalized the same way the production ones were
* I assumed this only applies to the `/manual` docs, but I'm second-guessing this. Is this template used for other sections of the docs, in which case this is not accurate?

Reply:
> I assumed this only applies to the /manual docs, but I'm second-guessing this. Is this template used for other sections of the docs, in which case this is not accurate?

Ok, I have a better understanding of how the theme is used, and I've confirmed that whatever I put here is also going to need to function for other sections of the c7 docs. Even though canonicals are out of scope on, say, https://docs.camunda.org/cawemo/latest/ and https://docs.camunda.org/security/, both of those use this theme, so this canonical URL needs to at-worst preserve the original value, and at-best _also_ rewrite it properly to remove the version. 

So I'll work on an updated regex to include in this PR.

Reply:
No, actually, I think this is exactly the Regex we want! 

I was thinking Hugo would emit an empty URL if the regex didn't match; instead it falls back to the original value. Which is perfect -- it means that anything in the `/manual` section of the docs will get rewritten to remove the version; any other sections will emit self-canonicals.

Which might not be exactly what we want for the SEO of the `/cawemo` section of the docs, but this work is not considering what is best for the Cawemo docs. 

If you feel that I should rewrite this regex so that it replaces an optional version instead of only applying to the `/manual` section, let me know -- otherwise I think this is correct as-is.

Reply:
> I included https:// at the beginning because I think we only care that this URL is accurate in production.

While testing locally, I realized that Hugo rewrites the configured URL to be `http://` even if you tell it to be `https://`. As a result, I updated the Regex on this PR so that the `s` is optional -- this leads to canonicals matching better between local & production. 

(With the `s` required, local would emit self-canonicals even when rendering the `/manual` section of the docs.)

Reply:
> I included https:// at the beginning because I think we only care that this URL is accurate in production.

While testing locally, I realized that Hugo rewrites the configured URL to be `http://` even if you tell it to be `https://`. As a result, I updated the Regex on this PR so that the `s` is optional -- this leads to canonicals matching better between local & production. 

(With the `s` required, local would emit self-canonicals even when rendering the `/manual` section of the docs.)

Reply:
> any other sections will emit self-canonicals.

Is this the same behavior that's already happening? Or is the current behavior no canonicals of any kind? Just want to make sure I'm clear if we are introducing some kind of negative behavior to the Cawemo docs.

Reply:
Self-canonicals would be new behavior. The layout currently emits no canonicals, for any docs. I think we'd be introducing a neutral behavior to the Cawemo docs -- if every version of a page has a self-canonical, I think that's no different than all of the versions having no canonical. (Theoretically......because crawlers do what crawlers do.)